tests/utilities/test_shuffle.py

```diff
@@ -21,8 +23,37 @@ def test_intra_node_chunk_shuffle():
 
 
 def test_associate_chunks_and_internals_to_ranks():
+    @dataclass
+    class Interval:
```

Comment:
Oh, at the time, I meant to use this in the Config and StreamingDataset, not in the test ;) But all good.